dash_canvas/package.json

```diff
@@ -18,7 +18,8 @@
         "build:js-dev": "webpack --mode development",
         "build:js": "webpack --mode production",
         "build:py": "node ./extract-meta.js src/lib/components > dash_canvas/metadata.json && copyfiles package.json dash_canvas && python -c \"import dash; dash.development.component_loader.generate_classes('dash_canvas', 'dash_canvas/metadata.json')\"",
-        "build:all": "npm run build:js && npm run build:js-dev && npm run build:py"
+        "build:all": "npm run build:js && npm run build:js-dev && npm run build:py",
+        "postbuild:all": "es-check es5 dash_canvas/*.js"
```

Comment:
Perhaps this would be a good opportunity to standardize on just `build` and `postbuild`, like we've done elsewhere?